examples/games/desk_toy.rs

```diff
@@ -20,7 +20,8 @@ fn main() {
     App::new()
         .add_plugins(DefaultPlugins.set(WindowPlugin {
             primary_window: Some(Window {
-                title: "Bevy Desk Toy".into(),
+              //  title: "Bevy Desk Toy".into(),
+                // adding title has a bug 
```

Comment:
This should be above the removed code, and say something more specific, like "MacOS does not let you set the title of transparent windows".